src/MusicStore/Components/AnnouncementComponent.cs

```diff
@@ -25,6 +26,13 @@ public IMemoryCache Cache
             set;
         }
 
+        [Activate]
+        public ISystemClock SystemClock
```

Comment:
@Eilon 
There is two identical system clock from Microsoft.AspNet.Authentication and  Microsoft.Framework.Caching.Memory.Infrastructure. 
@Praburaj suggested me to use one that under Caching. I think we should combine them at some point.